src/AFQMC/Memory/HIP/hip_arch.cpp

```diff
@@ -19,9 +19,9 @@
 #include "AFQMC/Memory/device_pointers.hpp"
 #include "mpi3/communicator.hpp"
 #include "mpi3/shared_communicator.hpp"
-#include "hipblas.h"
+#include <hipblas/hipblas.h>
 #include "hipsparse.h"
```

Comment:
No change in hipsparse?

Reply:
Valid point. We don't get any warnings but I will try it.

Reply:
updated